modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -42,7 +42,7 @@ trait ConsensusManager[F[_], Key, Artifact] {
   private[consensus] def facilitateOnEvent: F[Unit]
   private[consensus] def checkForStateUpdate(key: Key)(resources: ConsensusResources[Artifact]): F[Unit]
   private[sdk] def checkForStateUpdateSync(key: Key)(resources: ConsensusResources[Artifact]): F[Unit]
-
+  private[consensus] def registeredPeers: F[List[PeerId]]
```

Comment:
This is not a method that in any way "manages" consensus, therefore  I don't think it should be in `ConsensusManager`

Reply:
Should I introduce sth like `ConsensusService`?

Reply:
A private method in `ConsensusRoutes` would be enough I think.